web/static/js/reducers/questionnaire.js

```diff
@@ -108,19 +123,19 @@ const changeChoice = (state, action) => {
   }))
 }
 
-const autoComplete = (state, value) => {
+const autoComplete = (state, value, quiz: Questionnaire) => {
   let setted = false
 
   let smsValues = ''
   let ivrValues = ''
 
   state.forEach((step) => {
-    if (!setted) {
+    if ((step.type === 'multiple-choice') && !setted) {
       step.choices.forEach((choice) => {
         if (choice.value == value && !setted) {
           setted = true
-          smsValues = choice.responses['en'].sms.join(',')
-          ivrValues = choice.responses['en'].ivr.join(',')
+          smsValues = choice.responses[quiz.defaultLanguage].sms.join(',')
```

Comment:
This should probably be changed to check all prompts and responses in all languages in all modes.